students/geonwoo/users/views.py

```diff
@@ -35,3 +35,20 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message":"KEY_ERROR"},status=400)
             
+class LoginView(View):
+    def post(self, request):
+        try:
+            data  = json.loads(request.body)
+           
+            if not User.objects.filter(email = data['email']).exists():
+                return JsonResponse({"message":"INVALID_USER"},status=401)
+            
+            if not User.objects.filter(password = data['password']).exists():
+                return JsonResponse({"message":"INVALID_PASSWORD"},status=401)
+          
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+        
+            
+        return JsonResponse({"message": "SUCCESS"}, status=200)
```

Comment:
- except로 예외 처리한 경우 가장 아래로 내려주는 것이 가독성에 더 좋습니다.
- 불필요한 공백들은 제거해 주세요.
```suggestion
            return JsonResponse({"message": "SUCCESS"}, status=200)
        except KeyError:
            return JsonResponse({"message": "KEY_ERROR"}, status=400)
```